src/conduits/RwaInputConduit3.sol

```diff
@@ -36,14 +41,14 @@ import {GemJoinAbstract} from "dss-interfaces/dss/GemJoinAbstract.sol";
  */
 contract RwaInputConduit3 {
     /// @notice PSM GEM token contract address
-    DSTokenAbstract public immutable gem;
+    GemAbstract public immutable gem;
     /// @notice PSM contract address
     PsmAbstract public immutable psm;
 
     /// @dev This is declared here so the storage layout lines up with RwaInputConduit.
     DSTokenAbstract private __unused_gov;
```

Comment:
Since this is not used, instead of importing `DSTokenAbstract`, we could just declare this as an `address` and get rid of the dependency.

Reply:
done